src/main/java/am/itspace/overnight/config/WebSecurityConfig.java

```diff
@@ -0,0 +1,44 @@
+package am.itspace.overnight.config;
+import am.itspace.overnight.entity.RoleUser;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.security.config.annotation.authentication.builders.AuthenticationManagerBuilder;
+import org.springframework.security.config.annotation.web.builders.HttpSecurity;
+import org.springframework.security.config.annotation.web.configuration.EnableWebSecurity;
+import org.springframework.security.config.annotation.web.configuration.WebSecurityConfigurerAdapter;
+import org.springframework.security.core.userdetails.UserDetailsService;
+import org.springframework.security.crypto.password.PasswordEncoder;
+
+@EnableWebSecurity
+public class WebSecurityConfig extends WebSecurityConfigurerAdapter {
+
+    @Autowired
+    private UserDetailsService userDetailsService;
+
+    @Autowired
+    private PasswordEncoder passwordEncoder;
+
+    @Override
+    protected void configure(HttpSecurity http) throws Exception {
+        http
+                .csrf().disable()
+                .formLogin()
+                .and()
+                .logout()
+                .logoutSuccessUrl("/")
+                .and()
+                .authorizeRequests()
+                .antMatchers("/").permitAll()
+                .antMatchers("/").permitAll()
+                .antMatchers("/").hasAuthority(RoleUser.SELLER.name())
+                .antMatchers("/").permitAll()
+                .antMatchers("/").hasAuthority(RoleUser.CLIENT.name())
+                .anyRequest()
+                .permitAll();
```

Comment:
for first step we can permit any request to all types of users

 .csrf().disable()
                .formLogin()
                .and()
                .logout()
                .logoutSuccessUrl("/")
                .and()
                .authorizeRequests()
              //  .antMatchers("/").permitAll()
             //   .antMatchers("/").permitAll()
              //  .antMatchers("/").hasAuthority(RoleUser.SELLER.name())
             //   .antMatchers("/").permitAll()
             //   .antMatchers("/").hasAuthority(RoleUser.CLIENT.name())
                .anyRequest()
                .permitAll();